app/lib/features/super_invites/dialogs/redeem_dialog.dart

```diff
@@ -0,0 +1,157 @@
+    final info = ref.watch(superInviteInfoProvider(token));
+    return AlertDialog(
+      title: Text(L10n.of(context).redeem),
+      content: Container(
+        constraints: const BoxConstraints(
+          maxWidth: 500,
+        ),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            info.when(
+              data: (info) => renderInfo(context, ref, info),
+              error: (error, stackTrace) {
+                _log.severe(
+                  'Loading super invite failed: $token',
+                  error,
+                  stackTrace,
+                );
+                final errorStr = error.toString();
+                if (errorStr.contains('error: [404]')) {
+                  // Server doesn't yet support previewing
+                  return Text(
+                    L10n.of(context).superInvitesPreviewMissing(token),
+                  );
+                }
+                if (errorStr.contains('error: [403]')) {
+                  // 403 means we can't use that anymore
+                  return Text(
+                    L10n.of(context).superInvitesDeleted(token),
+                  );
+                }
+                return Text(L10n.of(context).loadingFailed(error));
+              },
+              loading: () => Skeletonizer(
+                child: Card(
+                  child: ListTile(
+                    leading: ActerAvatar(
+                      mode: DisplayMode.DM,
+                      avatarInfo: const AvatarInfo(
+                        uniqueId: 'nothing',
+                      ),
+                      size: 18,
+                    ),
+                    title: const Text('some random name'),
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      actionsAlignment: MainAxisAlignment.spaceEvenly,
+      actions: <Widget>[
+        OutlinedButton(
+          onPressed: () =>
+              Navigator.of(context, rootNavigator: true).pop(false),
+          child: Text(L10n.of(context).cancel),
+        ),
+        ActerPrimaryActionButton(
+          key: redeemConfirmKey,
+          onPressed: () => redeem(context, ref),
+          child: Text(L10n.of(context).redeem),
+        ),
+      ],
+    );
+  }
+
+  Widget renderInfo(BuildContext context, WidgetRef ref, SuperInviteInfo info) {
+    final displayName = info.inviterDisplayNameStr();
+    final userId = info.inviterUserIdStr();
+    return Padding(
+      padding: const EdgeInsets.all(15),
+      child: Card(
+        child: ListTile(
+          key: redeemInfoKey,
+          title: Text(
+            L10n.of(context).superInvitedBy(
+              displayName != null ? '$displayName ($userId)' : userId,
+            ),
+          ),
+          subtitle: Text(
+            L10n.of(context).superInvitedTo(info.roomsCount()),
```

Comment:
Is there any possibility to show room type bifurcation. Something Like `Abc user invites to you join 1 Space and 2 Chats`?

Reply:
not at this point - we are not exposing any details about the rooms other than the total count from the server yet.